src/k8s-engine/lima.ts

```diff
@@ -726,33 +726,19 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
       return;
     }
     await this.showSudoReason(explanations);
+    const singleCommand = commands.join('; ');
```

Comment:
We can still have `exec &> >(tee ${ logfile })` & `set -ex` in the command list (to get the logging), right?

Reply:
I'm not sure about it, which is why I left it out.

Running `bash -c 'CMD1;CMD2;CMD3…'`

is more like sourcing a file containing those commands than exec'ing them (e.g. `./run-script.sh`), and I don't like messing with the current session like that.

Although the "current session" is just a single `ssh` session to run the commands, so maybe... I'll have a look.

Reply:
Just added `ex` to the `/bin/sh` command -- no need to manage the output in a separate log file when the output gets echoed to `lima.log`